src/core/packages/overlays/browser-internal/src/flyout/flyout_service.tsx

```diff
@@ -128,14 +129,29 @@ export class FlyoutService {
           );
         };
 
+        const overlayMountWrapperStyle = css({
+          display: 'flex',
+          flexDirection: 'column',
+          height: '100%',
+          overflow: 'hidden',
+        });
+
+        // NOTE: The kbnOverlayMountWrapper className is used for allowing consumers to add additional styles
+        // that support drag-and-drop (padding, pointer styles). It is not used for internal styling.
```

Comment:
Hi @elastic/kibana-visualizations, could you help review this PR? The className is used for adding additional styles in `lens/public/trigger_actions/open_lens_config/helpers.scss`